docs/api/scheduler.md

```diff
@@ -85,6 +85,12 @@ Displays general information about the applications like used resources, queue n
 
 **Method** : `GET`
 
+**Query Params** : 
+
+1. queue=<fully qualified queue name\>
```

Comment:
is \> necessary? or just > is fine here?

Reply:
> is required to mark the close/end of param value. Backslash is required for escaping to avoid syntax error.